db/fixtures/miq_user_roles.yml

```diff
@@ -1075,6 +1087,9 @@
   - chargeback
   - catalog
   - cloud_tenant
+  - ems_configuration
+  - configuration_profile
+  - configured_system
```

Comment:
```suggestion
  - configured_systems
```

Reply:
this one should remain `configured_system` see https://github.com/ManageIQ/manageiq/pull/19949/files#diff-bccee92afbf0d3d195b7bcf343b1b6c4R4698